src/index.ts

```diff
@@ -76,7 +76,6 @@ import {
   linkMenuRegistry,
   otRegistry,
   rowMenuRegistry,
-  sheetMenuRegistry,
```

Comment:
Why is it not replaced by `getSheetMenuRegistry` ?

Reply:
I'd say taht since its not something that extensible, it didn't make that much sense to expose it. But I can export it if you think that my be useful.

Reply:
You're right